includes/blocks/class-sensei-progress-bar-block.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * File containing the Sensei_Progress_Bar_Block class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Progress_Bar_Block
+ */
+class Sensei_Progress_Bar_Block {
```

Comment:
I think we should name this `Course Progress (Block)` everywhere, since it's not a generic progress bar, but one specific for a course.

Reply:
Yep I was thinking about that too. Let's do this as part #3689 to avoid conflicts.